channels/us.m3u

```diff
@@ -3695,3 +3695,5 @@ https://rockentertainment-zoomoo-1.samsung.wurl.com/manifest/playlist.m3u8
 http://188.40.68.167/russia/disney/playlist.m3u8
 #EXTINF:-1 tvg-id="MeiGuoQieDaDianShi.us" tvg-name="美国切达电视" tvg-country="US" tvg-language="" tvg-logo="" group-title="",美国切达电视 (1080p)
 https://live.chdrstatic.com/cheddar/primary/4.m3u8
+#EXTINF:-1 tvg-id="PrivateTV.us" tvg-name="Private TV (Italia)" tvg-country="US" tvg-language="" tvg-logo="" group-title="XXX",Private TV (Italia)
```

Comment:
This is a channel from the Nederlands: https://www.lyngsat.com/tvchannels/nl/Private-TV.html

Reply:
Noted, Moving it when I can